src/pages/Pool/index.tsx

```diff
@@ -92,14 +94,30 @@ const MainContentWrapper = styled.main`
   flex-direction: column;
 `
 
+const ShowInactiveToggle = styled.div`
+  display: grid;
+  align-items: center;
+  justify-items: end;
+
+  grid-template-columns: 1fr auto;
+  grid-column-gap: 8px;
+  padding: 0 8px;
+`
+
 export default function Pool() {
   const { account } = useActiveWeb3React()
   const toggleWalletModal = useWalletModalToggle()
   const { t } = useTranslation()
   const theme = useContext(ThemeContext)
+  const [userHideClosedPositions, setUserHideClosedPositions] = useUserHideClosedPositions()
 
   const { positions, loading: positionsLoading } = useV3Positions(account)
 
+  const sortedPositions = positions?.sort((a, b) => (a.liquidity?.sub(b.liquidity ?? 0).isNegative() ? 1 : -1)) ?? []
```

Comment:
sorting by liquidity is pretty much arbitrary (depends on position range, and token decimals, has no relation to dollar value), token id is going to be more meaningful

Reply:
I was mostly just trying to put the closed positions at the end of the list. I'll update with sorting by tokenId for active and liquidity zero at the end